includes/rest-api/class-wp-job-manager-models-settings.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Declaration of our Settings Model
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Models_Settings extends WPJM_REST_Model_Declaration_Settings
```

Comment:
It seems like the logic that translates the settings array into the schema is mostly in `MT_Model_Declaration_Settings`. Do you think that will be used anywhere else besides WPJM?